src/components/screens/verifyMessage/verifyMessage.css

```diff
@@ -18,3 +18,9 @@
     vertical-align: middle;
   }
 }
+
+.result {
+  & h1 {
+    color: var(--color-maastricht-blue);
```

Comment:
💡  I was thinking if there is a better way to ensure that all headlines have a contrasting color to the background. 

I remember there were a lot of places where the headline was dark on dark by default, e.g.:
https://github.com/LiskHQ/lisk-desktop/blob/35a84736c4e405ea7a38762d79b2ebf9f8789270/src/components/screens/monitor/blocks/blocksOverview/blocksOverview.css#L4-L6


Maybe could be solved by setting `color: var(--color-maastricht-blue);` in the same place as where the dark background it set. In 
https://github.com/LiskHQ/lisk-desktop/blob/35a84736c4e405ea7a38762d79b2ebf9f8789270/src/components/toolbox/box/box.css#L7

or 
https://github.com/LiskHQ/lisk-desktop/blob/35a84736c4e405ea7a38762d79b2ebf9f8789270/src/app/app.css#L46

Reply:
It's good idea. I'll create a ticket to fix it after the beta release